tutorial.cpp

```diff
@@ -253,3 +258,32 @@ void Print(const Matrix& mat, ofstream& outFile) {
 void Print(const SubMatrix& sm, ofstream& outFile) {
 	outFile << sm << std::endl;
 }
+
+
+
+// The test for the Reader class.
+// @return int - 1 if failed, 0 if passed.
+int TestReader() {
+	int status = 1;
+
+	std::string testFileName = "current/ReaderTest.out";
+	std::ofstream outFile(testFileName);
+	if (!outFile.is_open()) {
+		std::cout << "FAILURE: Cannot open " << testFileName << "!" << std::endl;
+		return 1;
+	}
+
+	try{

```

Comment:
Add another test to check data that cannot be converted to int and another that has mismatched column numbers.